pkg/skaffold/schema/latest/config.go

```diff
@@ -123,7 +123,13 @@ type TagPolicy struct {
 type ShaTagger struct{}
 
 // GitTagger *beta* tags images with the git tag or commit of the artifact's workspace.
-type GitTagger struct{}
+type GitTagger struct {
+	// Variant determines the behavior of the git tagger. Valid variants are
```

Comment:
Will this doc string render well in the consumers of this doc?

Reply:
it's not the best, maybe we could remove the dashes?
multiline comments get merged into a single line - you can always check with `make preview-docs`! 
https://github.com/GoogleContainerTools/skaffold/pull/1902/files#diff-4db1e0a21e170b019f0d05230514a5ccR808
http://35.236.122.247:1313/docs/references/yaml/

Reply:
> you can always check with `make preview-docs`!

So true...
I agree, the dashes really don't help. But I just realized that I can wrap the enum values with backticks, which should help.